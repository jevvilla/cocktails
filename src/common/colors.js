export default {
  background: '#41AFC7',
  white: '#fff',
  black: '#000',
  whiteOpacity: 'rgba(255, 255, 255, 0.5)',
};
