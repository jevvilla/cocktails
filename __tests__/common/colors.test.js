import colors from '../../src/common/colors';

describe('colors', () => {
  test('should match snapshot', () => {
    expect(colors).toMatchSnapshot();
  });
});
