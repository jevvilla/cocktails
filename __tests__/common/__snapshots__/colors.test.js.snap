// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors should match snapshot 1`] = `
Object {
  "background": "#41AFC7",
  "black": "#000",
  "white": "#fff",
  "whiteOpacity": "rgba(255, 255, 255, 0.5)",
}
`;
